### ISSUE TITLE
Datapoint list icons improvements

### ISSUE THREAD

**Issue by valllllll2000** [akvo/akvo-flow-mobile#1602]
### Scope
This issue is about improving the datapoint list icons which represent the different state a datapoint has: Saved, Exported and Uploaded.

### Background
Currently here is what we have for the datapoint list

<img src="https://user-images.githubusercontent.com/923280/75770860-197f7280-5d49-11ea-9478-088c85114a98.png" width="200" />

Which is not the most estetically appealing layout. 
With small changes to the code we could improve it significantly.
There is already an issue to remove the alternative background https://github.com/akvo/akvo-flow-mobile/issues/1593

## Problem
How can we improve it but still make sure the users know when the datapoint was uploaded to the dashboard correctly or if it's only exported or saved?

Here is a design by @Kiarii 
<img width="263" alt="Screenshot 2020-03-06 at 8 40 49" src="https://user-images.githubusercontent.com/923280/76065744-820a6180-5f8c-11ea-9caf-de4573f9ae84.png">

**Comment by valllllll2000** [comment]
Here is one example of how it could look with icons:
<img src="https://user-images.githubusercontent.com/923280/76090353-b3e6ec80-5fbb-11ea-9325-5ce4b6854f17.png" width="200" />

**Comment by Kiarii** [comment]
@valllllll2000 do we have a workflow of all possible statuses (and their descriptions) that a datapoint can be in - not sure if there was an issue about this before - , it would be nice to see the bigger picture and think of iconography that might be meaningful before settling on the solution..  A solution might also mean that we rethink the statuses i.e. if it would make sense to do so..

**Comment by Kiarii** [comment]
There will also be a question of where to position the status icons on the list component; there is a number of options and it would be nice if we can consider these to determine the most suitable..

**Comment by valllllll2000** [comment]
@Kiarii 
Here are the different datapoint status
* Saved: datapoint have not been submitted, user can still edit it (Orange color)
* Submitted: datapoint was submitted so it can no longer be edited but it was not uploaded to dashboard (Orange color)
* Uploaded: datapoint has been sent to dashboard or this datapoint has been downloaded from dashboard. (Green color)

**Comment by valllllll2000** [comment]
Update: this is the current state of the UI
<img src="https://user-images.githubusercontent.com/923280/79440921-e0d4e900-7fd6-11ea-80a4-8d51310f51f0.png" width="200" />

**Comment by janagombitova** [comment]
We had this discussion a while back if you remember and we dismissed the colourless icons because of what we learned from the user research: 
* Users use the icons to quickly scan the list and see the state of their submissions. 
* Having the icons small meant "searching" more 
* Having the icons less prominent and colourless was not appreciated as then they really needed to take more effort in checking the state of the submissions. 

Why do users want to know the state of submissions: 
* the **saved** state requires immediate action to finish the data input and submit
* the **submitted** state is usually understood as 'something is wrong with my connection' and then users opt to 'force' upload by clicking on the `upload` option in the hamburger menu. 

I am not saying we should not revamp the icons, but we should not make them so different that we use the value they provide today.
